prov/util/src/util_mr_cache.c

```diff
@@ -320,9 +318,11 @@ int ofi_mr_cache_search(struct ofi_mr_cache *cache, const struct fi_mr_attr *att
 			struct ofi_mr_entry **entry)
 {
 	struct ofi_mr_info info;
+	struct ofi_mem_monitor *monitor;
+	bool flush_lru;
 	int ret;
-	struct ofi_mem_monitor *monitor = cache->monitors[attr->iface];
 
+	monitor = cache->monitors[attr->iface];
```

Comment:
Sneaky change lol

Reply:
Wasn't meant as sneaky, just a readability cleanup that's not worth doing as a separate patch while I'm in the function